spec/mixin/content/arguments/receiving/output-libsass.css

```diff
@@ -0,0 +1,28 @@
+with-defaults nothing-passed {
+  arg1: value1;
+  arg2: value2;
+}
+with-defaults partial-override {
+  arg1: value1;
+  arg2: other2;
+}
+with-defaults total-override {
+  arg1: other1;
+  arg2: other2;
+}
+
+with-splat nothing-passed {
+  positional: ();
+  keywords: ();
+}
+with-splat positional-passed {
+  positional: value1, value2, value3;
+  keywords: ();
+}
+with-splat named-passed {
+  keywords: (arg1: value1, arg2: value2);
```

Comment:
Changed some output due to an existing issue with inspect'ing arglists that isn't related to this feature.